frida_tools/tracer.py

```diff
@@ -35,9 +35,9 @@ def process_builder_arg(option, opt_str, value, parser, method, **kwargs):
                     type='string', action='callback', callback=process_builder_arg, callback_args=(pb.include_modules,))
             parser.add_option("-X", "--exclude-module", help="exclude MODULE", metavar="MODULE",
                     type='string', action='callback', callback=process_builder_arg, callback_args=(pb.exclude_modules,))
-            parser.add_option("-i", "--include", help="include FUNCTION", metavar="FUNCTION",
+            parser.add_option("-i", "--include", help="include [MODULE]![FUNCTION]", metavar="FUNCTION",
```

Comment:
```suggestion
            parser.add_option("-i", "--include", help="include [MODULE!]FUNCTION", metavar="FUNCTION",
```

Should we do it something like this (here and below) to communicate what is optional?